Production_scripts/Process_extracts/Produce_extract_tests/extract_chi_ltc_tests.R

```diff
@@ -0,0 +1,93 @@
+####################################################
+# Name of file - extract_chi_ltc_tests.R
+# Original Authors - Bateman McBride
+# Original Date - August 2022
+# Written/run on - RStudio Server
+# Version of R - 3.6.1
+# Description - Produce tests for source linkage files:
+#               LTC file received from IT.
+#####################################################
+
+year <- "1920"
+
+new_data <- haven::read_sav(get_ltcs_path(year))
+
+# Find and flag any duplicate chis and chi/postcode combinations
+duplicates <- new_data %>%
+  dtplyr::lazy_dt() %>%
+  dplyr::group_by(chi) %>%
+  dplyr::mutate(duplicate_chi = dplyr::n() > 1) %>%
+  dplyr::ungroup() %>%
+  dplyr::group_by(chi, postcode) %>%
+  dplyr::mutate(duplicate_chi_postcode = dplyr::n() > 1) %>%
+  dplyr::ungroup() %>%
+  dplyr::summarise(dplyr::across(c("duplicate_chi", "duplicate_chi_postcode"), sum)) %>%
+  dplyr::ungroup() %>%
+  tibble::as_tibble() %>%
+  tidyr::pivot_longer(
+    cols = tidyselect::everything(),
+    names_to = "measure",
+    values_to = "value"
+  )
```

Comment:
This is good and working as expected. Thanks for showing us how to use `dtplyr::lazy_dt()` we have been able to implement the use of this function in the social care datasets and this will also be very useful going forward when the code gets more complicated :)

Only comment - is there a way to trim this down without having to do multiple group() and ungroup()? Just for improving the readability of the code

Reply:
I think there would have been, but the need to group repeatedly has been superseded by other suggestions